### DISCUSSION TITLE
Css failing to load?

### DISCUSSION THREAD

**Discussion by Jaudr3yy** [community/community#21804] [Other Feature Feedback, Questions, & Ideas]
<p>Hello! This may be pretty simple, however, I am new to this. I have a css file names style.css and a main html file called index.html. The issue is that my css isn’t loading to my website (<a href="https://bloodywalsh.github.io/crownbot/" rel="nofollow noopener">https://bloodywalsh.github.io/crownbot/</a>) even though it is setup. Here are some links: <br>
<strong>Css</strong>  <a href="https://github.com/Bloodywalsh/crownbot/blob/master/style.css" rel="nofollow noopener">https://github.com/Bloodywalsh/crownbot/blob/master/style.css</a> <br>
<strong>Html </strong> <a href="https://github.com/Bloodywalsh/crownbot/blob/master/index.html" rel="nofollow noopener">https://github.com/Bloodywalsh/crownbot/blob/master/index.html</a></p>
<p>Any ideas why it isn’t loading?</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>I don’t see a link to style.css inside index.html.  E.g.:</p>
<pre><code>&lt;head&gt;
&lt;link rel="stylesheet" type="text/css" href="style.css"&gt;
... rest of head ...
&lt;/head&gt;
</code></pre>

**Comment by hassan647** [comment]
<p>you forget to add link in your html file for external css stylesheet so You can add it by using the following syntax</p>
<p>&lt;link rel=“stylesheet” type=“text/css” href=“mystyle.css”&gt;<br>
&lt;style&gt;</p>

**Comment by srinivasthutika** [comment]
<p>Is this is working for you ??</p>
<p>If yes can you please share the solution.</p>
